includes/admin/class-amp-plugins.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class AMP_Plugins
+ *
+ * @package AMP
+ * @since   2.2.2
+ */
+
+/**
+ * Class to add
+ */
+class AMP_Plugins {
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @return void
+	 */
+	public function init() {
+
+		if ( ! is_admin() ) {
+			return;
+		}
+
+		add_filter( 'plugin_row_meta', [ $this, 'filter_plugin_row_meta' ], 10, 3 );
+	}
+
+	/**
+	 * Get list of suppressed plugins.
+	 *
+	 * @return array List of of suppressed plugins.
+	 */
+	protected static function get_suppressed_plugins() {
+
+		static $suppressed_plugins = [];
+
+		if ( empty( $suppressed_plugins ) ) {
+			$suppressed_plugins = AMP_Options_Manager::get_option( 'suppressed_plugins' );
+		}
+
+		return $suppressed_plugins;
+	}
```

Comment:
I appreciate the desire for efficiency, but adding a `static` variable will complicate testing. The performance gains here would be minimal.